core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/WorkspaceApiService.scala

```diff
@@ -189,6 +189,13 @@ trait WorkspaceApiService extends UserInfoDirectives {
           }
         }
       } ~
+      path("workspaces" / Segment / Segment / "fileTransfers") { (workspaceNamespace, workspaceName) =>
+        get {
+          complete {
+            workspaceServiceConstructor(userInfo).listPendingFileTransfersForWorkspace(WorkspaceName(workspaceNamespace, workspaceName)).map(pendingTransfers => StatusCodes.OK -> pendingTransfers)
```

Comment:
Just for my own Scala education, what does `.map(pendingTransfers => StatusCodes.OK -> pendingTransfers)` do?

Reply:
That part is taking the result of the `listPendingFileTransfersForWorkspace` `Future`, unboxes the result contained in the `Future` (which I've named `pendingTransfers`), and then transforming the result into the `StatusCodes.OK -> pendingTransfers` pair, which is then re-boxed into the `Future` and fed to the outer `complete` akka-http directive, which returns the status code and marshals `pendingTransfers` into a JSON response. 

In this particular case, this extra step is not actually needed because `OK` is the default response code for a successful `Future`, so the result of `listPendingFileTransfersForWorkspace` would have been returned anyway with an OK status code. But I still like to do it because it makes the code more readable IMO. If I wanted this to return a status code other than OK, I'd definitely need to do this step.

Reply:
Thanks for the explanation.